### PR TITLE
Example how to retrieve cpu and memory usage in python

### PR DESCRIPTION
**Pull request by acamacho82**

### CODE REVIEW

File: content/python/get_virtual_guest_usage_records.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+title: "CPU and Memory usage in Virtual servers"
+description: "How the portal page retrieves the memory and cpu usage"
+date: "2019-04-10"
+classes:
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Metric_Tracking_Object"
+tags:
+    - "metric"
+    - "metrics"
+    - "virtual server"
```
Comment (allmightyspiff) [inline content/python/get_virtual_guest_usage_records.md]: lets go with `virtualguest` as the tag. (so remove `virtual guest`)
and remove the `metrics` tag, no need for the pluralization.

File: content/python/get_virtual_guest_usage_records.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        return averages
+
+    def get_cpu_usage(self, guest_id, start_date, end_date):
+        metric_object_id = self._get_metric_tracking_object(guest_id)
+
+        # build the SoftLayer_Container_Metric_Data_Type array
+        valid_types = []
+        # set range according number of cpu items you want to retrieve the metric data
+        for i in range(16):
```
Comment (allmightyspiff) [inline content/python/get_virtual_guest_usage_records.md]: couldn't you get this number `16` from the virtual guest object itself?

File: content/python/get_virtual_guest_usage_records.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    for record in cpu_records: print(record)
+
+    print("\nCPU AVERAGES:")
+    print(cpu_averages)
+
+    # print records and memory usage
+    print("\nMEMORY USAGE RECORDS:")
+    for record in memory_records: print(record)
+    
+    # there is only 1 memory and its value must be divided by 2 to convert it to GB
```
Comment (allmightyspiff) [inline content/python/get_virtual_guest_usage_records.md]: `# there is only 1 memory and its value must be divided by 2^30 to convert it to GB`

File: content/python/get_virtual_guest_usage_records.md (hunk 1 above)
Comment (acamacho82) [reply content/python/get_virtual_guest_usage_records.md]: Done

File: content/python/get_virtual_guest_usage_records.md (hunk 2 above)
Comment (acamacho82) [reply content/python/get_virtual_guest_usage_records.md]: that is a very good idea, I will retrieve the `startCpus` using masks

**Pull request merged**